### ISSUE TITLE
Unable to run examples locally SyntaxError: `Unexpected identifier in examples/views/hello.ejs while compiling ejs`

### ISSUE THREAD

**Issue by lb-** [hotwired/stimulus#588]
I am unable to use the local `examples` via `yarn start` - not sure what to do differently or if there is a step I have missed.

## Steps to reproduce

1. Fork the Stimulus project locally and `cd` into that folder
2. Use `fnm` to run Node `16` or Node `14` (tried both
3. Running Yarn v 1.22.17
4. Run `yarn install`
5. Run `yarn start`
6. Open a browser at `http://localhost:9000/`
7. Throws error


## Stack trace

```
SyntaxError: Unexpected identifier in stimulus/examples/views/hello.ejs while compiling ejs

If the above error is not helpful, you may want to try EJS-Lint:
https://github.com/RyanZim/EJS-Lint
Or, if you meant to create an async function, pass `async: true` as an option.
    at new Function (<anonymous>)
    at Template.compile (.../examples/node_modules/ejs/lib/ejs.js:673:12)
    at Object.compile (.../examples/node_modules/ejs/lib/ejs.js:398:16)
    at handleCache (.../examples/node_modules/ejs/lib/ejs.js:235:18)
    at tryHandleCache (.../examples/node_modules/ejs/lib/ejs.js:274:16)
    at View.exports.renderFile [as engine] (.../examples/node_modules/ejs/lib/ejs.js:491:10)
    at View.render (.../examples/node_modules/express/lib/view.js:135:8)
    at tryRender (.../examples/node_modules/express/lib/application.js:640:10)
    at Function.render (.../examples/node_modules/express/lib/application.js:592:3)
    at ServerResponse.render (.../examples/node_modules/express/lib/response.js:1012:7)
```

## Full console stack trace

```
yarn start
yarn run v1.22.17
$ concurrently "npm:watch" "npm:start:examples"
[start:examples] 
[start:examples] > @hotwired/stimulus@3.1.0 start:examples
[start:examples] > cd examples && yarn install && node server.js
[start:examples] 
[watch] 
[watch] > @hotwired/stimulus@3.1.0 watch
[watch] > rollup -wc
[watch] 
[1/4] Resolving packages...
success Already up-to-date.
[watch] rollup v2.53.1
[watch] bundles src/index.js → dist/stimulus.umd.js, dist/stimulus.js...
[start:examples] Listening on http://localhost:9000/
[start:examples] Browserslist: caniuse-lite is outdated. Please run:
[start:examples] npx browserslist@latest --update-db
[start:examples] 
[start:examples] Why you should do it regularly:
[start:examples] https://github.com/browserslist/browserslist#browsers-data-updating
[start:examples] ℹ ｢wdm｣: Hash: b0e52bf7819538edf9af
[start:examples] Version: webpack 4.46.0
[start:examples] Time: 1413ms
[start:examples] Built at: 04/10/2022 7:54:56 pm
[start:examples]   Asset     Size  Chunks             Chunk Names
[start:examples] main.js  396 KiB    main  [emitted]  main
[start:examples] Entrypoint main = main.js
[start:examples] [../dist/stimulus.js] 59.3 KiB {main} [built]
[start:examples] [./controllers/clipboard_controller.js] 1.93 KiB {main} [built]
[start:examples] [./controllers/content_loader_controller.js] 2.83 KiB {main} [built]
[start:examples] [./controllers/hello_controller.js] 1.77 KiB {main} [built]
[start:examples] [./controllers/slideshow_controller.js] 2.4 KiB {main} [built]
[start:examples] [./index.js] 651 bytes {main} [built]
[start:examples] [./node_modules/@babel/runtime/helpers/assertThisInitialized.js] 297 bytes {main} [built]
[start:examples] [./node_modules/@babel/runtime/helpers/classCallCheck.js] 274 bytes {main} [built]
[start:examples] [./node_modules/@babel/runtime/helpers/createClass.js] 674 bytes {main} [built]
[start:examples] [./node_modules/@babel/runtime/helpers/defineProperty.js] 367 bytes {main} [built]
[start:examples] [./node_modules/@babel/runtime/helpers/getPrototypeOf.js] 425 bytes {main} [built]
[start:examples] [./node_modules/@babel/runtime/helpers/inherits.js] 586 bytes {main} [built]
[start:examples] [./node_modules/@babel/runtime/helpers/possibleConstructorReturn.js] 447 bytes {main} [built]
[start:examples] [./node_modules/@babel/runtime/helpers/setPrototypeOf.js] 395 bytes {main} [built]
[start:examples] [./node_modules/turbolinks/dist/turbolinks.js] 37.6 KiB {main} [built]
[start:examples]     + 1 hidden module
[start:examples] ℹ ｢wdm｣: Compiled successfully.
[watch] created dist/stimulus.umd.js, dist/stimulus.js in 2.6s
[watch] bundles src/index.js → dist/stimulus.min.js...
[start:examples] ℹ ｢wdm｣: Compiling...
[start:examples] ℹ ｢wdm｣: Hash: b0e52bf7819538edf9af
[start:examples] Version: webpack 4.46.0
[start:examples] Time: 172ms
[start:examples] Built at: 04/10/2022 7:54:58 pm
[start:examples]   Asset     Size  Chunks  Chunk Names
[start:examples] main.js  396 KiB    main  main
[start:examples] Entrypoint main = main.js
[start:examples] [../dist/stimulus.js] 59.3 KiB {main} [built]
[start:examples] [./controllers/clipboard_controller.js] 1.93 KiB {main}
[start:examples] [./controllers/content_loader_controller.js] 2.83 KiB {main}
[start:examples] [./controllers/hello_controller.js] 1.77 KiB {main}
[start:examples] [./controllers/slideshow_controller.js] 2.4 KiB {main}
[start:examples] [./index.js] 651 bytes {main}
[start:examples] [./node_modules/@babel/runtime/helpers/assertThisInitialized.js] 297 bytes {main}
[start:examples] [./node_modules/@babel/runtime/helpers/classCallCheck.js] 274 bytes {main}
[start:examples] [./node_modules/@babel/runtime/helpers/createClass.js] 674 bytes {main}
[start:examples] [./node_modules/@babel/runtime/helpers/defineProperty.js] 367 bytes {main}
[start:examples] [./node_modules/@babel/runtime/helpers/getPrototypeOf.js] 425 bytes {main}
[start:examples] [./node_modules/@babel/runtime/helpers/inherits.js] 586 bytes {main}
[start:examples] [./node_modules/@babel/runtime/helpers/possibleConstructorReturn.js] 447 bytes {main}
[start:examples] [./node_modules/@babel/runtime/helpers/setPrototypeOf.js] 395 bytes {main}
[start:examples] [./node_modules/turbolinks/dist/turbolinks.js] 37.6 KiB {main}
[start:examples]     + 1 hidden module
[start:examples] ℹ ｢wdm｣: Compiled successfully.
[watch] created dist/stimulus.min.js in 2.3s
[start:examples] SyntaxError: Unexpected identifier in /.../stimulus/examples/views/hello.ejs while compiling ejs
[start:examples] 
[start:examples] If the above error is not helpful, you may want to try EJS-Lint:
[start:examples] https://github.com/RyanZim/EJS-Lint
[start:examples] Or, if you meant to create an async function, pass `async: true` as an option.
[start:examples]     at new Function (<anonymous>)
[start:examples]     at Template.compile (/.../stimulus/examples/node_modules/ejs/lib/ejs.js:673:12)
[start:examples]     at Object.compile (/.../stimulus/examples/node_modules/ejs/lib/ejs.js:398:16)
[start:examples]     at handleCache (/.../stimulus/examples/node_modules/ejs/lib/ejs.js:235:18)
[start:examples]     at tryHandleCache (/.../stimulus/examples/node_modules/ejs/lib/ejs.js:274:16)
[start:examples]     at View.exports.renderFile [as engine] (/.../stimulus/examples/node_modules/ejs/lib/ejs.js:491:10)
[start:examples]     at View.render (/.../stimulus/examples/node_modules/express/lib/view.js:135:8)
[start:examples]     at tryRender (/.../stimulus/examples/node_modules/express/lib/application.js:640:10)
[start:examples]     at Function.render (/.../stimulus/examples/node_modules/express/lib/application.js:592:3)
[start:examples]     at ServerResponse.render (/.../stimulus/examples/node_modules/express/lib/response.js:1012:7)
[start:examples] SyntaxError: Unexpected identifier in /.../stimulus/examples/views/hello.ejs while compiling ejs
[start:examples] 
[start:examples] If the above error is not helpful, you may want to try EJS-Lint:
[start:examples] https://github.com/RyanZim/EJS-Lint
[start:examples] Or, if you meant to create an async function, pass `async: true` as an option.
[start:examples]     at new Function (<anonymous>)
[start:examples]     at Template.compile (/.../stimulus/examples/node_modules/ejs/lib/ejs.js:673:12)
[start:examples]     at Object.compile (/.../stimulus/examples/node_modules/ejs/lib/ejs.js:398:16)
[start:examples]     at handleCache (/.../stimulus/examples/node_modules/ejs/lib/ejs.js:235:18)
[start:examples]     at tryHandleCache (/.../stimulus/examples/node_modules/ejs/lib/ejs.js:274:16)
[start:examples]     at View.exports.renderFile [as engine] (/.../stimulus/examples/node_modules/ejs/lib/ejs.js:491:10)
[start:examples]     at View.render (/.../stimulus/examples/node_modules/express/lib/view.js:135:8)
[start:examples]     at tryRender (/.../stimulus/examples/node_modules/express/lib/application.js:640:10)
[start:examples]     at Function.render (/.../stimulus/examples/node_modules/express/lib/application.js:592:3)
[start:examples]     at ServerResponse.render (/.../stimulus/examples/node_modules/express/lib/response.js:1012:7)
^C
[start:examples] npm run start:examples exited with code SIGINT                                                                                                                                                                                                              
[watch] npm run watch exited with code SIGINT
```

**Comment by marcoroth** [comment]
Thanks for opening this as a separate issue. I can actually replicate this on my machine.

**Comment by lb-** [comment]
This local Dev server had the same issues for me pre-eslint, I first encountered it when I did the other PR (about a month ish ago).

**Comment by marcoroth** [comment]
Root cause for this was #562, this is now resolved via #600.

**Comment by lb-** [comment]
Thank you @marcoroth  - working great for me now
